theseus/utils/examples/motion_planning/motion_planner.py

```diff
@@ -172,25 +171,64 @@ def __init__(
                 )
             )
 
+
+class MotionPlanner:
```

Comment:
Since this abstraction is the same as before does this PR cause a breaking change in the API?

Reply:
It does because the order of args changed in the constructor.